### DISCUSSION TITLE
Stuck on empty spectrogram at initial setup

### DISCUSSION THREAD

**Discussion by harrymnel** [Nachtzuster/BirdNET-Pi#232] [Q&A]
![Screenshot 2024-11-11 at 14 12 21](https://github.com/user-attachments/assets/0a9c65b2-2ded-4d99-8e75-8f84561d16d8)
My BirdNet is stuck on this empty spectrogram from the initial setup and has not changed after 30 minutes of running. The live audio is showing that the microphone is working?

**Comment by Nachtzuster** [comment]
Could you open an issue for this?
please add the install log and a 'View Log' screenshot

**Comment by alexbelgium** [comment]
Have you checked for microphone gain is not 0 ? I already saw someone with the same issue somewhere. https://github.com/mcguirepr89/BirdNET-Pi/wiki/Adjusting-your-sound-card
